source/backend/api/Areas/Documents/DocumentRelationshipController.cs

```diff
@@ -120,39 +124,6 @@ public IActionResult GetRelationshipDocuments(DocumentRelationType relationshipT
             }
         }
 
-        /// <summary>
```

Comment:
yeah, I think the document controller makes more sense then the relationship controller.

Reply:
The clear indication of this was that the relationshiptype was not being used at all in the call